openfisca_core/legislations.py

```diff
@@ -220,7 +220,7 @@ def compact_dated_node_json(dated_node_json, code = None, instant = None, parent
         assert instant is not None
         name = u'.'.join((parent_codes or []) + [code]) \
             if code is not None \
-            else None
+            else ''
```

Comment:
I'd rather prefer keeping the `None` value since it represents more explicitly the absence of value.

Then fix the crash where it happens, but it's perhaps more difficult.